cmd/kube-controller-manager/app/mizarcontrollers.go

```diff
@@ -20,10 +20,10 @@ package app
 
 import (
 	"net/http"
-	"time"
 
 	arktos "k8s.io/arktos-ext/pkg/generated/clientset/versioned"
 	"k8s.io/arktos-ext/pkg/generated/informers/externalversions"
+	"k8s.io/client-go/dynamic"
```

Comment:
can we use explicitly typed client?

Reply:
Dynamic is introduced here for CRD objects only. Typed client won't work.